eureka/src/main/java/com/linecorp/armeria/client/eureka/EurekaEndpointGroup.java

```diff
@@ -74,6 +76,13 @@
  */
 public final class EurekaEndpointGroup extends DynamicEndpointGroup {
 
+    /**
+     * Instance info attributeKey.
+     */
+    @UnstableApi
+    public static final AttributeKey<InstanceInfo> INSTANCE_INFO = AttributeKey.valueOf(
+            EurekaEndpointGroup.class, "instanceInfo");
```

Comment:
We prefer adding a helper class to hide the implementation detail(key name).
Should we add:
```java
// Need a better name 😅
public final class EurekaEndpointInstance {
    // Share the key between this class and EurekaEndpointGroup
    static final AttributeKey<InstanceInfo> INSTANCE_INFO = AttributeKey.valueOf(
            EurekaEndpointInstance.class, "instanceInfo");

    public InstanceInfo get(Endpoint endpoint) {
         ...
    }
}
```

Reply:
How about EurekaEndpointUtils or InstanceInfoUtils?

Reply:
Sounds good to me 👍

Reply:
How about prefixing `Eureka` to `InstanceInfoUtils` like `EurekaInstanceInfoUtils`?

Reply:
Q) Has this comment been addressed?